package.json

```diff
@@ -54,10 +40,23 @@
     "gulp-uglify": "^1.2.0",
     "gulp-util": "^3.0.6",
     "install": "^0.1.8",
+    "mozilla-tabzilla": "^0.4.0",
+    "mustache": "^2.1.3",
+    "node-bourbon": "^4.2.3",
+    "node-neat": "^1.7.2",
+    "node-sass": "^3.2.0",
+    "normalize.css": "^3.0.3",
     "run-sequence": "^1.1.2",
+    "through2": "^2.0.0",
     "vinyl-buffer": "^1.0.0",
     "vinyl-source-stream": "^1.1.0"
   },
+  "devDependencies": {
+    "babel-eslint": "^4.1.0",
+    "eslint-config-airbnb": "^0.0.8",
+    "eslint-config-idea-town": "^1.2.0",
+    "gulp-eslint": "^1.0.0"
+  },
```

Comment:
So... Not sure if this is 100% right. I basically moved everything from devDeps into deps, and then moved everything "eslint" into devDeps since they aren't required for building or deploying and not something we'd want/need if we deployed to prod.